components/textbox/src/_mixin.scss

```diff
@@ -291,13 +306,85 @@ $utrecht-support-prince-xml: false !default;
   --utrecht-textbox-value-max-length: 4;
 }
 
+@mixin utrecht-textbox--voorletters-size {
+  /* "Voorletters" depends on how long your name is and after research of the longest name in the Netherlands it can be about 8 to 9 characters.
+     For now im using the longest name source suggestion: 8 */
+  --utrecht-textbox-value-max-length: 8;
+}
+
 @mixin utrecht-textbox--voorvoegsel-size {
   /* "Tabel 36 Voorvoegselstabel" by "Rijksdienst voor Identiteitsgegevens" has a list
    * of all allowed values. The longest value is 10 characters.
    * The class name is in Dutch because it is a Dutch concept. */
   --utrecht-textbox-value-max-length: 10;
 }
 
+@mixin utrecht-textbox--voornamen-size {
+  /* Longest name in the Netherlands is 50 characters. Suggesting: 40 */
+  --utrecht-textbox-value-max-length: 40;
+}
+
+@mixin utrecht-textbox--vestigingnummer-size {
+  /* Typically 9 or lower digits */
+  --utrecht-textbox-value-max-length: 10;
+}
+
+@mixin utrecht-textbox--bsn-size {
+  /* BSN is 8 or 9 characters. Suggest: 5 */
+  --utrecht-textbox-value-max-length: 5;
+}
+
+@mixin utrecht-textbox--email-size {
+  /* Emails can very with how long they are. Suggesting max */
+  --utrecht-textbox-value-max-length: 40;
+}
+
+@mixin utrecht-textbox--geboortedatum-size {
+  /* 2 (for the day) + 1 (for the separator, usually a dash or slash) + 2 (for the month) + 1 (for the separator)
+   + 4 (for the year) = 9 characters (if you include separators). Or "DDMMYY" For now im testing: 5 */
+  --utrecht-textbox-value-max-length: 5;
+}
+
+@mixin utrecht-textbox--geboorteplaats-size {
+  /* testing 25 */
+  --utrecht-textbox-value-max-length: 25;
+}
+
+@mixin utrecht-textbox--achternaam-size {
+  /* Longest lastname in the Netherlands is 58 characters. Suggesting: 40 */
+  --utrecht-textbox-value-max-length: 40;
+}
+
+@mixin utrecht-textbox--kvk-size {
+  /* In the Netherlands a KVK number consists of 8 digits */
+  --utrecht-textbox-value-max-length: 4;
```

Comment:
Waarom niet 8?

Reply:
`max-length: 4;` is precies goed voor 8 karakters
`--utrecht-textbox-value-max-length: 4;`

<img width="500" alt="Screenshot 2024-11-26 at 13 54 56" src="https://github.com/user-attachments/assets/f855568b-6fa2-4896-b1d1-6a0aec276fbb">

`max-length: 8;` lijkt mij te groot voor niks
`--utrecht-textbox-value-max-length: 8;`

<img width="500" alt="Screenshot 2024-11-26 at 13 54 46" src="https://github.com/user-attachments/assets/70b1c4de-daff-439d-b4a3-1596a0b6838f">

Misschien is de berekening per karakter niet correct?